esteban-madrigal-ak/training-assignments
# Task03 one piece website

# Pull Request
## Description
This branch contains the code for the one piece application with mobile, tablet and desktop mode, there is a style folder where you can find the style files, also there is the html and js files corresponding to the app

## Type of change

## How Has This Been Tested?
I test the functionality of the application with help of an extension called live server that helps to see the changes of the html and how it looks.

- install the extension
- open the html file you want to test on vsc
- select the go live option on the bottom panel
- it will open the web browser showing the html
- every change on the html or css is going to be showed on the browser window
